### ISSUE TITLE
11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: VoiceOver is  not reading the roles for "Azure Functions","Microsoft" ,"740657","Star Rating","Repository" and "License" as links

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2688]
### Actual Result:
VoiceOver is not reading the roles for "Azure Functions","Microsoft" ,"740657","Star Rating","Repository" and "License" links

### Expected Result:
VoiceOver should be reading the roles for "Azure Functions","Microsoft" ,"740657","Star Rating","Repository" and "License" links instead of just reading them as "Empty Groups"

### GitHubTags:
#Azure Tools for VS Code;#A11ySev1;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG1.3.1;#MAC; #ScreenReader;#VoiceOver;#Element:Link;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Start VoiceOver
2. Open Visual studio code
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab till "Azure Functions","Microsoft" ,"740657","Star Rating","Repository" and "License" links and check whether VoiceOver is reading the roles as links or just reading them as empty groups

### User Impact:
Screen Reader users will not know the exact purpose of these controls if Voice Over is just reading them as Empty groups instead of reading asa links

### Attachment for Reference:
[Bug_2688_VoiceOver is not reading the roles for "Azure Functions","Microsoft" ,"740657","Star Rating","Repository" and "License" links.zip](https://github.com/microsoft/vscode-azurefunctions/files/6001536/Bug_2688_VoiceOver.is.not.reading.the.roles.for.Azure.Functions.Microsoft.740657.Star.Rating.Repository.and.License.links.zip)

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
